src/filterdiff.js

```diff
@@ -1,31 +1,29 @@
-import { spawn } from "child_process";
```

Comment:
github is making this file look weird but fyi i didn't manually change anything; this was just the output of standard --fix.